controllers/nutanixcluster_controller.go

```diff
@@ -230,6 +238,24 @@ func (r *NutanixClusterReconciler) reconcileNormal(rctx *nctx.ClusterContext) (r
 	return reconcile.Result{}, nil
 }
 
+func (r *NutanixClusterReconciler) reconcileProject(rctx *nctx.ClusterContext) error {
```

Comment:
we also need to modify machine controller to add teh vm to project if initially vm was not in project when cluster was created but later was added thru yaml modification

Reply:
per @yannickstruyf3 this will be handled thru https://jira.nutanix.com/browse/KRBN-5342